pages/linux/gedit.md

```diff
@@ -11,10 +10,10 @@
 
 `gedit {{file1 file2 ...}}`
 
-- Display list of supported encodings:
+- Open a text file with a specific encoding:
 
-`gedit --list-encodings`
+`gedit --encoding={{UTF-8}} {{path/to/file}}`
 
-- Open a text file with the specified encoding:
+- Display a list of the supported encodings:
```

Comment:
If we want to use `the` here, it might sound better like this:
```suggestion
- Display a list of the encodings supported:
```

Reply:
Makes sense, thanks! That was clearly my Portuguese creeping in! 😄

Reply:
After re-reading both variants, I think I prefer the the-less version:

```suggestion
- Display a list of supported encodings:
```

WDYT?

Reply:
The current version ("supported encodings") sounds better than @sbrl's suggestion to me. Removing the article altogether though also seems like a good idea.

Reply:
Yeah, let's remove the article.

Reply:
Done :)